test/functional/rpc_rawtransaction.py

```diff
@@ -200,20 +201,20 @@ def createrawtransaction_tests(self):
         # sequence number out of range
         for invalid_seq in [-1, 4294967296]:
             inputs = [{'txid': TXID, 'vout': 1, 'sequence': invalid_seq}]
-            outputs = {self.nodes[0].getnewaddress(): 1}
+            outputs = {getnewdestination()[2]: 1}
             assert_raises_rpc_error(-8, 'Invalid parameter, sequence number is out of range',
                                     self.nodes[0].createrawtransaction, inputs, outputs)
         # with valid sequence number
         for valid_seq in [1000, 4294967294]:
             inputs = [{'txid': TXID, 'vout': 1, 'sequence': valid_seq}]
-            outputs = {self.nodes[0].getnewaddress(): 1}
+            outputs = {getnewdestination()[2]: 1}
```

Comment:
nit, not sure how obvious it is to readers that `getnewdestination()[2]` is an address without looking up the function, so (here and just above) this might be clearer
```diff
-            outputs = {getnewdestination()[2]: 1}
-            rawtx = self.nodes[0].createrawtransaction(inputs, outputs)
+            address = getnewdestination()[2]
+            rawtx = self.nodes[0].createrawtransaction(inputs=inputs, outputs={address: 1})
```

Reply:
Fixed, thanks!
From a quick `git grep` it seems to me the usage of `getnewdestination` is clear in the other tests:
```
test/functional/feature_coinstatsindex.py:    getnewdestination,
test/functional/feature_coinstatsindex.py:        reorg_blocks = self.generatetoaddress(index_node, 2, getnewdestination()[2])
test/functional/interface_rest.py:    getnewdestination,
test/functional/interface_rest.py:        txid, _ = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=getnewdestination()[1], amount=int(0.1 * COIN))
test/functional/interface_rest.py:        txid, _ = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=getnewdestination()[1], amount=int(0.1 * COIN))
test/functional/p2p_filter.py:    getnewdestination,
test/functional/p2p_filter.py:        block_hash = self.generatetoscriptpubkey(getnewdestination()[1])
test/functional/p2p_filter.py:        self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=getnewdestination()[1], amount=7 * COIN)
test/functional/p2p_filter.py:            txid, _ = self.wallet.send_to(from_node=self.nodes[0], scriptPubKey=getnewdestination()[1], amount=7 * COIN)
test/functional/p2p_filter.py:            block_hash = self.generatetoscriptpubkey(getnewdestination()[1])
test/functional/rpc_createmultisig.py:    getnewdestination,
test/functional/rpc_createmultisig.py:            self.final = getnewdestination()[2]
test/functional/rpc_createmultisig.py:        pk0 = getnewdestination()[0].hex()
test/functional/rpc_createmultisig.py:        pk1 = getnewdestination()[0].hex()
test/functional/rpc_createmultisig.py:        pk2 = getnewdestination()[0].hex()
test/functional/rpc_rawtransaction.py:    getnewdestination,
test/functional/rpc_rawtransaction.py:            address = getnewdestination()[2]
test/functional/rpc_rawtransaction.py:            address = getnewdestination()[2]
test/functional/rpc_rawtransaction.py:        address = getnewdestination()[2]
test/functional/rpc_rawtransaction.py:        address2 = getnewdestination()[2]
test/functional/rpc_scantxoutset.py:    getnewdestination,
test/functional/rpc_scantxoutset.py:        pubk1, spk_P2SH_SEGWIT, addr_P2SH_SEGWIT = getnewdestination("p2sh-segwit")
test/functional/rpc_scantxoutset.py:        pubk2, spk_LEGACY, addr_LEGACY = getnewdestination("legacy")
test/functional/rpc_scantxoutset.py:        pubk3, spk_BECH32, addr_BECH32 = getnewdestination("bech32")
test/functional/test_framework/wallet.py:def getnewdestination(address_type='bech32'):
```